docs/2.0/docs/patcher/guides/report.md

```diff
@@ -1,37 +1,9 @@
 # Patcher Report
 
-Starting in `0.4.2`, Patcher supports two modes: **interactive mode** and **non-interactive mode**.
-
-In interactive mode, the user can browse the discovered module dependencies.
+Patcher supports **non-interactive mode** for `patcher report`.
```

Comment:
This is a bit of a funny way to phrase things - most CLI tools are non-interactive, so saying "non-interactive mode" is like saying "Patcher CLI is a CLI."